tests/tests/test-nonce.ts

```diff
@@ -1,12 +1,15 @@
 import { expect } from "chai";
-import { step } from "mocha-steps";
 
 import { createAndFinalizeBlock, describeWithMoonbeam, customRequest } from "./util";
 import { Keyring } from "@polkadot/keyring";
 import { GENESIS_ACCOUNT, GENESIS_ACCOUNT_PRIVATE_KEY, TEST_ACCOUNT } from "./constants";
 
 describeWithMoonbeam("Moonbeam RPC (Nonce)", `simple-specs.json`, (context) => {
-  step("get nonce", async function () {
+  before(async () => {
+    // For some reason fees are not well estimated otherwise
+    await createAndFinalizeBlock(context.polkadotApi);
```

Comment:
This is because of the delay I think, not because of block being created.
Can you retry now that I added an extra delay in the node start ?

Reply:
Does not help, if I skip the first test the second one fails without the block creation :(

Reply:
It overestimates the gas needed for the transfer

Reply:
Ok keep it for now.
We should report that as a bug

Reply:
It might be because we use a maximum gas limit based on the block gas limit which is set only on the first block